### DISCUSSION TITLE
PII request review. What is a timeline at the moment ?

### DISCUSSION THREAD

**Discussion by alexartisans** [amzn/selling-partner-api-models#3307] [General]
Opened a case on April 8th. Still "under review". Is that normal ? Or should I start worrying ?
We only need so we can automate and ship our own custom MFN orders.
It is crazy that even with buy Amazon labels , SP API does NOT allow you to download custom information from the order without PII nor purchase labels through them (that one I'm not sure about though).

**Comment by alexartisans** [comment]
We got that reviewed and denied. Wonderful.
Apparently I can NOT download custom data for MY orders. 
My app didn't OFFER substantially different functionality...
However THERE IS NOTHING on the market that offers this functionality at the moment.
Nothing that I can find anyway....
This seems crazy...

**Reply by phstc** [reply]
Did you eventually get approved?

**Comment by alexartisans** [comment]
Nope. They removed MWS PII info on top of it. "Nothing special". Cant even
get to automate my own orders...
What a ,ess

On Sun, Sep 11, 2022 at 6:15 PM Pablo Cantero ***@***.***>
wrote:

> Did you eventually get approved?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/amzn/selling-partner-api-docs/discussions/2205#discussioncomment-3621315>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AYZEXXMJFJLYUAD2BJGD32LV5ZKXLANCNFSM5TWRKRRA>
> .
> You are receiving this because you authored the thread.Message ID:
> <amzn/selling-partner-api-docs/repo-discussions/2205/comments/3621315@
> github.com>
>


-- 
Alex Pershin
CEO Of Basic Fundamentals
Artisans Online, LLC
305-923-4718

**Comment by ross-worth** [comment]
Can we get a response here, we are also trying to get approved but it is taking weeks and the response wasn't entirely helpful. Seems like this needs to be a case for anti-competition lawyers at the moment.

**Comment by jamesquinton** [comment]
We are getting the same response when applying for restricted access. We have provided all documentation and detailed answers. 

What concerns us most is this announcement - https://developer-docs.amazon.com/sp-api/changelog/important-notice-discontinued-support-lwa-tokens-can-no-longer-retrieve-pii-in-selling-partner-api. 

We need to be able to continue collecting orders after August 31st for our internal reporting. Ideally we would like to be able to do fulfillment via the API as well, but that is not our #1 priority.

**Reply by jamesquinton** [reply]
@jevoniuk are you able to clarify? I notice you posted the announcement in this repo.

**Reply by jimmy-ross** [reply]
@jamesquinton @jevoniuk do you know if this announcement means that no order data will be accessible at all via the current LWA token process? even the basic get order and get order items operations? or its just the PII data related to orders that will require the tokens API/ RDT? 

In other words, do you need PII approval even to get the most basic order information (order ID etc) or just for the parts that include actual PII (e.g. Order Items Buyer info)?

**Reply by jamesquinton** [reply]
@jimmy-ross-xapix This is what I am also trying to understand - in the original announcement, this is not clear. I've opened a support ticket and and pushing for an answer via all available channels. I'll let you know if I hear anything.

**Reply by jimmy-ross** [reply]
thanks @jamesquinton, yeah i also asked once ages ago and couldnt get a clear answer. also just opened a new specific discussion about it: https://github.com/amzn/selling-partner-api-docs/discussions/3214
